static/app/utils/replays/playback/providers/replayPlayerEventsContext.tsx

```diff
@@ -10,9 +10,11 @@ export function ReplayPlayerEventsContextProvider({
   replay,
 }: {
   children: React.ReactNode;
-  replay: ReplayReader;
+  replay: ReplayReader | null;
```

Comment:
it it really null sometimes, or is this to make the types happy?

Reply:
it's a pain, but might be possible to chase around the types and remove a bunch of `| null`

Reply:
I'll do this in a follow up PR!